docs/topics/modules/assigning_an_area.adoc

```diff
@@ -10,6 +10,6 @@ You can group Work Items by type or functionality by assigning relevant Areas to
 .Procedure
 . Click the btn:[Plan] tab to view a list of Work Items for your Space.
 . Double-click a Work Item to view its details.
-. Click the *Area* text box to view a list of Areas to assign to the space. By default, your Root Area is assigned to a new Work Item.
+. Click the *Area* field to view a list of Areas to assign to the space. By default, your Root Area is assigned to a new Work Item.
```

Comment:
I just tested the instructions and it look good!

Reply:
Hi @petra-sargent. The Plan button is the one on the top of every page (except your OSIO home page). It should be the only "Plan" button no matter what you're doing in OSIO. Is this clear enough, or maybe I can add a "From the top of the page" to direct users? WDYT?

Reply:
Hey Misha, 
This is something I discussed earlier with Robert too. Maybe a good time to come to discuss and come to a common understanding.
For me "Analyze, Plan, and Create' seem to be tabs and not buttons. For instance when you click Plan the view it generates is more like a Tab. Buttons to me are 'Create work item' on the dashboard. Thoughts?

Reply:
What does Todd use in his doc?

Reply:
Tab.

Reply:
@mishaone "From the top of the page" does make it more clear.  Analyze, Plan and create do seem more like tabs to me rather than buttons.

Reply:
I'm not sure if there's much point in saying, for example, "Click the Plan tab" since stylistically we would just say "Click Plan". The markup btn is what I'm using anytime there is an element to click (element rather than just text to click). Do we want to change this across the board and what to? "Click *Plan* to..." perhaps?

Reply:
Clarification, since md renders asterisks as italics. In the comment above, the potential change was "Click * Plan * to..." so that the name of the tab would appear bold.

Reply:
"Click **Plan** to " also seems consistent with our practice of highlighting all UX text in bold. So imo this seems to be good. This also makes mentioning its position, `on top` etc redundant I suppose, since there is only one Plan tab.

Reply:
> I'm not sure if there's much point in saying, for example, "Click the Plan tab" since stylistically we would just say "Click Plan".

I wouldn't be so sure about this. In order to make the instructions unambiguous, I would consider "Click the Plan tab". Or better, I think we can assume some common knowledge of UI principles, so we could just say: "Switch to the Plan tab". That way, we're making it clear that *tabs* are involved,  and we're telling the user how to proceed. It can be expected that users know that in order to get to some other tab, they need to click on said tab's 'tab'.

> The markup btn is what I'm using anytime there is an element to click (element rather than just text to click).

@mishaone, for the sake of clarity, I would suggest using the `btn[]` syntax *only* for real buttons. I.e. not for all clickable GUI elements, but strictly for buttons. So, for example, when describing the comment input box here on GH, there would be two buttons (Cancel, Comment) and two tabs (Write, Preview).

Reply:
> I wouldn't be so sure about this. In order to make the instructions unambiguous, I would consider "Click the Plan tab". Or better, I think we can assume some common knowledge of UI principles, so we could just say: "Switch to the Plan tab". That way, we're making it clear that tabs are involved, and we're telling the user how to proceed. It can be expected that users know that in order to get to some other tab, they need to click on said tab's 'tab'.

The style recommendations comes from IBM (I can add the specifics tomorrow at work from my physical copy), which is generally the final say in these things. Personally, I agree with not having a clunkier sounding "Click the X thing" and would vote for a reasonable compromise of "At the top of the page, click X" and refer to the actual component as the tab (e.g. "In the Planner tab, ...") rather than referring to the UI element as a tab in the click instruction. That's my two cents, but happy to go with whatever the majority opinion is.

Reply:
From IBM> Label: Refer to a tab by its label, followed by tab. Example: In the Display notebook, click the Screen Saver tab.
When we mention Tab, the need to mention location ("At the top of the page") seems redundant, and also favors brevity.

Reply:
I'm a little uncertain about calling them out as tabs at all, to be honest. Standard UI tabs should have a clear tab on top attached to the content below, with other tabs sort of grayed out or otherwise clearly not selected, which doesn't seem to be the case here (maybe it should be?).
Here are some examples off the top of a Google image search for what I would call tabs: https://www.google.com.au/search?q=UI+tab&source=lnms&tbm=isch&sa=X&ved=0ahUKEwi1zd-W7tXXAhWErJQKHehlDdcQ_AUICigB&biw=1920&bih=1107

tl:dr: I don't dispute IBM's advice on documenting tabs, I would be uncertain about calling the top row of options in OSIO tabs at all, since they don't look the way tabs usually do in UIs and may therefore just be confusing.

Reply:
@Preeticp and @rkratky, this has been open a while, so I'd suggest merging it and splitting off the discussion about style changes (or just a task to change this across the board) to a new issue. WDYT?

Reply:
@mishaone, I believe this PR has already been merged.